### DISCUSSION TITLE
Option or a shortcut to pause or turn off selected nodes

### DISCUSSION THREAD

**Discussion by T3chRomanc3r** [comfyanonymous/ComfyUI#1024] [Ideas]
Sometimes it's needed to check output of just one node without activating the rest connected nodes. 

In this example I want to quickly render different seeds to find more interesting composition as a base for generations, without activating 2 other following KSamplers:
![Screenshot 2023-07-30 065722](https://github.com/comfyanonymous/ComfyUI/assets/45420556/8c57b4cc-2ceb-4f0e-842a-1c97a3037b1a)

**Comment by ztalien** [comment]
Maybe "ctrl+M".
https://blenderneko.github.io/ComfyUI-docs/Interface/Shortcuts/

**Comment by photigy** [comment]
Its called Fast Bypass (or Mute) node, I think its from Mixlab

**Comment by BrechtCorbeel** [comment]
Or an option to run them in isolation?

**Comment by kauboiBibappu** [comment]
Ctrl+B. Or right-click on the group of nodes and select 'Bypass Group Nodes.' To enable them again, select 'Set Group Nodes to Always'.

**Comment by NeoAnthropocene** [comment]
This is an old post but for the ones who can reach out the same place for an answer here is the quick tip:

1. Install [rgthree-comfy](https://github.com/rgthree/rgthree-comfy) custom node (I think it's a must have for an average ComfyUI user)
2. Activate the "Show fast toggles in Group headers".
3. Enjoy easy controls specifically on your groups

![image](https://github.com/user-attachments/assets/5c3c01d6-64f1-4d3b-8d3a-e268c0e6de1a)

![image](https://github.com/user-attachments/assets/d8d61fc5-6f57-4267-9092-834fb95ef534)

The most useful part is; you can run a specific group by just clicking the run icon on your groups. That feature saves me a lot of time.
![image](https://github.com/user-attachments/assets/5b89d0ef-6a2e-45a5-97e2-4fce439f13d8)
